util-taglib/src/META-INF/liferay-ui.tld

```diff
@@ -700,7 +700,7 @@
 			<type>boolean</type>
 		</attribute>
 		<attribute>
-			<description>Whether to disable the namespace for the component. The default value is <![CDATA[<code>false</code>]]>.</description>
+			<description>Whether to disable the namespace for the component. The default value is <![CDATA[<code>false</code>]]>. Deprecated as of 7.4.0, replaced by the attribute <![CDATA[<code>useNamespace</code>]]>.</description>
```

Comment:
Why are we replacing `disableNamespace`, false by default, with `useNamespace`, true by default? Isn't this the exact same thing?

Reply:
Hey @markocikos,
Yes it is, however there are other tags which follow the pattern of using `useNamespace` (see [BaseInputTag](https://github.com/liferay/liferay-portal/blob/master/util-taglib/src/com/liferay/taglib/aui/base/BaseInputTag.java#L230-L232)).  Since it accomplishes the same thing, and `disableNamespace` is only used in the `InputdateTag`, I figured it would be best to switch it out.

Reply:
OK, makes sense :+1: